docs/src/devdocs/elements.md

```diff
@@ -11,6 +11,7 @@ of faces `M`.
 ```@docs
 Ferrite.vertices(::Ferrite.AbstractCell)
 Ferrite.edges(::Ferrite.AbstractCell)
+Ferrite.reference_faces(::Ferrite.AbstractRefShape)
```

Comment:
I don't see where this is used in this PR?

Reply:
I checked it. It's not used directly but it's used to get `nfaces` for reference shapes which is used in `create_face_quad_rule`. It'll be used in the other PR in a more direct way, but I think it's necessary here too